templates/terraform/.github/workflows/auto-context.yml

```diff
@@ -1,55 +1,55 @@
 name: "auto-context"
 on:
   schedule:
-    # Update context.tf nightly
-    - cron: "0 3 * * *"
+  # Update context.tf nightly
+  - cron:  '0 3 * * *'
 
 jobs:
   update:
     if: github.event_name == 'schedule'
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+    - uses: actions/checkout@v2
 
-      - name: Update context.tf
-        shell: bash
-        id: update
-        env:
-          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
-        run: |
-          if [[ -f context.tf ]]; then
-            echo "Discovered existing context.tf! Fetching most recent version to see if there is an update."
-            curl -o context.tf -fsSL https://raw.githubusercontent.com/cloudposse/terraform-null-label/master/exports/context.tf
-            if git diff --no-patch --exit-code context.tf; then
-              echo "No changes detected! Exiting the job..."
-            else
-              echo "context.tf file has changed. Update examples and rebuild README.md."
-              make init
-              make github/init/context.tf
-              make readme/build
-              echo "::set-output name=create_pull_request=true"
-            fi
+    - name: Update context.tf
+      shell: bash
+      id: update
+      env:
+        GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+      run: |
+        if [[ -f context.tf ]]; then
+          echo "Discovered existing context.tf! Fetching most recent version to see if there is an update."
+          curl -o context.tf -fsSL https://raw.githubusercontent.com/cloudposse/terraform-null-label/master/exports/context.tf
+          if git diff --no-patch --exit-code context.tf; then
+            echo "No changes detected! Exiting the job..."
           else
-            echo "This module has not yet been updated to support the context.tf pattern! Please update in order to support automatic updates."
+            echo "context.tf file has changed. Update examples and rebuild README.md."
```

Comment:
We have modules that do not use null label. I don't know how I feel about this because this is just updating one file and moving to context pattern is a lot more work. What is the benefit from this?

Reply:
This is the automatic update of `context.tf` you approved a week ago. All I did was revert whitespace changes from #265 and restore the cron job quotes.

Reply:
I missed that we introduced this 2nd type of PR for context. Not keen for it and it's potential to generate more PRs that we cannot look into and PRs that won't help in any implementation

Reply:
View the file, not the diff. All this does for modules that do not have `context.tf` already is output a message in the action log, not open a PR.